hub/management/commands/import_foe_groups.py

```diff
@@ -1,83 +1,65 @@
 from django.conf import settings
 
 import pandas as pd
-from tqdm import tqdm
 
-from hub.models import Area, AreaData, DataSet
+from hub.models import DataSet
 
-from .base_importers import BaseLatLongImportCommand
+from .base_importers import BaseConstituencyGroupListImportCommand
 
 
-class Command(BaseLatLongImportCommand):
+class Command(BaseConstituencyGroupListImportCommand):
     help = "Import data about FOE groups by constituency"
 
-    data_file = settings.BASE_DIR / "data" / "foe_groups.csv"
+    data_file = settings.BASE_DIR / "data" / "foe_groups_wmc.csv"
     message = "Importing FOE groups data"
     uses_gss = False
 
     defaults = {
         "data_type": "json",
         "category": "movement",
         "subcategory": "groups",
-        "label": "Active Friends of the Earth groups",
-        "description": "Descriptions of active Friends of the Earth groups by constituency.",
+        "label": "Friends of the Earth local action groups",
+        "description": "",
         "source_label": "Data from Friends of the Earth.",
-        "release_date": "November 2022",
+        "release_date": "January 2024",
         "source": "https://friendsoftheearth.uk/",
```

Comment:
We could probably link to https://friendsoftheearth.uk/about/what-are-local-action-groups, as that actually explains what the Local Action Groups are, and contains a link to search/browse groups on a map.